src/hotspot/share/utilities/resizeableResourceHash.hpp

```diff
@@ -85,19 +85,44 @@ class ResizeableResourceHashtable : public ResourceHashtableBase<
                                      K, V, ALLOC_TYPE, MEM_TYPE, HASH, EQUALS>;
   using Node = ResourceHashtableNode<K, V>;
   NONCOPYABLE(ResizeableResourceHashtable);
+
+  // Calculate next "good" hashtable size based on requested count
+  int calculate_resize(bool use_large_table_sizes) const {
+    const int resize_factor    = 2.0;     // by how much we will resize using current number of entries
```

Comment:
Nit: extra spaces brefore the '=' sign.
Q: Why is a FP constant assigned to the integer variable?